docs/pages/access-graph/getting-started-on-prem.mdx

```diff
@@ -0,0 +1,130 @@
+</Notice>
+
+## Prerequisites
+
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted Enterprise">
+
+- A running Teleport Enterprise cluster v14.3.4 or later.
+- An updated `license.pem` with Teleport Policy enabled.
+- Docker version v(=docker.version=) or later.
+- A PostgreSQL database server v14 or later.
+  - Access Graph needs a dedicated [database](https://www.postgresql.org/docs/current/sql-createdatabase.html) to store its data.
+    The user that TAG connects to the database with needs to be an owner of this database, or have similar broad permissions:
+    at least the `CREATE TABLE` privilege on the `public` schema, and the `CREATE SCHEMA` privilege.
+  - Amazon RDS for PostgreSQL is supported.
+- A TLS certificate for the Access Graph service
+- The node running the Access Graph service must be reachable from Teleport Auth and Proxy services.
+
+</TabItem>
+</Tabs>
+
+## Step 1/3. Set up the Teleport Access Graph service
+
+You'll need a copy of your Teleport cluster's host certificate authority (CA) on the machine that hosts the Access Graph service.
+TAG requires incoming connections to be authenticated via host certificates that the host CA issues to Auth and Proxy services.
+
+The host CA can be retrieved and saved into a file in one of the following ways:
+
+<Tabs>
+<TabItem label="Via curl">
+```console
+$ mkdir /etc/access_graph
+$ curl 'https://<Var name="teleport.example.com" />/webapi/auth/export?type=tls-host' > /etc/access_graph/teleport_host_ca.pem
+```
+</TabItem>
+
+<TabItem label="Via tctl">
+```console
+$ mkdir /etc/access_graph
+$ tsh login --proxy=<Var name="teleport.example.com" />
+$ tctl get cert_authorities --format=json \
+    | jq -r '.[] | select(.spec.type == "host") | .spec.active_keys.tls[].cert' \
+    | base64 -d > /etc/access_graph/teleport_host_ca.pem
+```
+</TabItem>
+</Tabs>
+
+Then, create a configuration file for the TAG service, similar to this:
+
+```yaml
+backend:
+  postgres:
+    # This uses the PostgreSQL connection URI format, see https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING-URIS
+    # A stricter `sslmode` value is strongly recommended,
+    # e.g. `sslmode=verify-full&sslrootcert=/etc/access_graph/my_postgres_ca.crt`.
+    # For a full reference on possible parameters see https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS
+    connection: postgres://access_graph_user:mypassword@db.example.com:5432/access_graph_db?sslmode=require
+
+    # When running on Amazon RDS, IAM auth via credentials set in the environment can be used as follows:
+    # iam:
+    #   aws_region: us-west-2
+
+# IP address (optional) and port for the TAG service to listen to.
+# This is the default value. This key can be omitted to listen on port 50051 on all interfaces.
+address: ":50051"
+
+tls:
+  # File paths of PEM-encoded TLS certificate and private key for the TAG server.
+  cert: /etc/access_graph/tls.crt
+  key: /etc/access_graph/tls.key
```

Comment:
This guide doesn't mention anything about how those certificates should be generated. Here is my proposal https://github.com/gravitational/access-graph/issues/445